src/components/tasks/card/index.tsx

```diff
@@ -121,7 +132,51 @@ const Card: FC<Props> = ({
       </span>
       )
   }
- 
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const { requestPromise } = fetch({ url: SELF_URL });
+        const { data } = await requestPromise;
+        const userRoles = {
+          adminUser: data.roles?.admin,
+          superUser: data.roles?.super_user,
+        };
+        const { adminUser, superUser } = userRoles;
+        setIsUserAuthorized(!!adminUser || !!superUser);
```

Comment:
what are we trying to do here?

Reply:
I guess we are converting both the values to boolean explicitly and checking if the user is admin or superuser only then he/she needs to be authorized